desktop/Makefile

```diff
@@ -158,13 +158,11 @@ node_modules/%:
 node_modules: package.json
 	@$(NPM) prune
 	@$(NPM) install
-	@touch node_modules
```

Comment:
Sorry I can't remember what the reason is. If it seems to work and nothing else has a problem then removing it is probably ok, and it's easy to add back if something does happen.